spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -489,6 +499,34 @@ class Analyzer(tableUtils: TableUtils,
     }
   }
 
+  // For groupBys validate if the timestamp provided produces some values
+  // if all values are null this should be flagged as an error
+  def runTimestampCheck(df: DataFrame, sampleFraction: Double = 0.1): String = {
```

Comment:
Could we also check if the timestamps are in milliseconds level? All the computation in the codebase is based on millisecond. 
Sometimes, users will use second or nanosecond, which will lead to null output values.

Reply:
I can look into it. Though since the data-type is just Bigint, we can't be sure what the user actually intends to enter in general use-cases. If someone wants to use `1720000000` (2024-07-03 if in seconds ), is also valid millisec (1970-01-20), if using some historic data set. 

One thing that might be do-able is that we check if assumed milliseconds between some "reasonable timeframe" like:
Jan 1, 1970 00:00:00 UTC - Jan 1, 2100 00:00:00 UTC 
(assuming chronon will be around for ~100 years!)

Reply:
@pengyu-hou - please take a look at a possible solution.

Reply:
Somehow I missed this message. The approach works.